src/lambda_telemetry_shipper/configuration.py

```diff
@@ -8,6 +8,18 @@ def parse_env(env_name: str, default: Optional[str] = None) -> Optional[str]:
     return os.environ.get(env_name, default)
 
 
+def parse_env_to_bool(env_name: str, default: bool) -> bool:
+    try:
+        if parse_env(env_name) == 'True':
```

Comment:
Can you please make it slightly easier to use, and modify it to by case insensitive?
```
if parse_env(env_name, default="").lower() == "true":
```

Reply:
Thanks, good idea.

Done.